collections/ansible_collections/cloudkit/config_as_code/roles/aap/vars/controller.yml

```diff
@@ -0,0 +1,216 @@
+    verbosity: 0
+
+# Create cluster fulfillment and confg-as-code inventories. They are the same
+# for the moment, but spliting them will allow us to diverge easily in the
+# future.
+controller_inventories:
+  - name: "{{ aap_prefix }}-cluster-fulfillment"
+    description: "Cluster fulfilment inventory"
+    organization: "{{ aap_organization_name }}"
+  - name: "{{ aap_prefix }}-config-as-code"
+    description: "Config-as-code inventory"
+    organization: "{{ aap_organization_name }}"
+
+controller_inventory_sources:
+  - name: "{{ aap_prefix }}-cluster-fulfillment-is"
+    organization: "{{ aap_organization_name }}"
+    source: scm
+    source_project: "{{ aap_prefix }}"
+    source_path: "inventory/localhost.yml"
+    inventory: "{{ aap_prefix }}-cluster-fulfillment"
+    overwrite: true
+    overwrite_vars: true
+    update_cache_timeout: 0
+  - name: "{{ aap_prefix }}-config-as-code-is"
+    organization: "{{ aap_organization_name }}"
+    source: scm
+    source_project: "{{ aap_prefix }}"
+    source_path: "inventory/localhost.yml"
+    inventory: "{{ aap_prefix }}-config-as-code"
+    overwrite: true
+    overwrite_vars: true
+    update_cache_timeout: 0
+
+controller_schedules:
+  # Sync project every 10min to get the latest updates from the git repository
+  - name: "{{ aap_prefix }}-sync"
+    description: "Periodic sync job for {{ aap_project_name }} project"
+    organization: "{{ aap_organization_name }}"
+    unified_job_template: "{{ aap_prefix }}"
+    rrule: "DTSTART:20250331T144500Z RRULE:FREQ=MINUTELY;INTERVAL=10"
+  # Run config-as-code template every hour to apply new changes, if any
+  - name: "{{ aap_prefix }}-config-as-code"
+    description: "Periodic run of config-as-code"
+    organization: "{{ aap_organization_name }}"
+    unified_job_template: "{{ aap_prefix }}-config-as-code"
+    rrule: "DTSTART:20250331T144500Z RRULE:FREQ=HOURLY;INTERVAL=1"
+
+# Create custom execution environments
+controller_execution_environments:
+  - name: "{{ aap_prefix }}-ee"
+    organization: "{{ aap_organization_name }}"
+    image: "{{ aap_ee_image }}"
+    pull: always
+
+# Customize pod templates in order to inject the configuration and the
+# credentials when the template are run, that way Kubernetes is the source
+# of truth for the secrets, and we don't need to sync credentials into AAP
+controller_instance_groups:
+  # For cluster-fulfillment use cases, we want to allow KubeAPI access to the
+  # cluster the pod works on, so we mount the credentials for the cloudkit
+  # ServiceAccount. All other credentials (AWS, Openstack, ...) that would be
+  # required by the playbooks are expected to be specified in the
+  # (prefix)-cluster-fulfilment-ig secret.
+  - name: "{{ aap_prefix }}-cluster-fulfillment-ig"
+    is_container_group: true
+    pod_spec_override: |
+      apiVersion: v1
+      kind: Pod
+      metadata:
+        namespace: fulfillment-aap
+        labels:
+          ansible_job: ''
```

Comment:
What does setting the `ansible_job` label to an empty string accomplish here?

Reply:
I added it because it's already there in the default template, but I'm not sure if it's required